ale_linters/ruby/rubocop.vim

```diff
@@ -25,7 +25,8 @@ function! ale_linters#ruby#rubocop#Handle(buffer, lines) abort
 endfunction
 
 function! ale_linters#ruby#rubocop#GetCommand(buffer) abort
-    return 'rubocop --format emacs --force-exclusion '
+    return ale#Var(a:buffer, 'ale_ruby_rubocop_executable')
```

Comment:
You should apply `ale#Escape` to this value here, in case the path to the executable contains spaces, or whatever.